.docs/angular-meteor/packages/tutorial-step-diff-compiler/plugin.js

```diff
@@ -22,6 +22,14 @@ function multiPatchHandler(compileStep) {
   });
 }
 
+function prepareSummary(message) {
```

Comment:
@barbatus are those changes should go in the https://github.com/meteor/tutorial-tools/ packages?